diff --git a/dist/import.mjs b/dist/import.mjs
index 5616b78dfc7cc8d9e30fbfa8c1eef95e25197b2c..36cb122af5755522ef5f7d4b9f61ccf39078f690 100644
--- a/dist/import.mjs
+++ b/dist/import.mjs
@@ -1,4 +1,5 @@
-import {useMessageFormatter as $345fc832e5b22baa$export$ec23bf898b1eed85} from "./private/useMessageFormatter.js";
+import pkg from "./private/useMessageFormatter.js";
+const {useMessageFormatter: $345fc832e5b22baa$export$ec23bf898b1eed85} = pkg;
 import {I18nProvider as $1697a7ad3db7b79f$re_export$I18nProvider, useLocale as $1697a7ad3db7b79f$re_export$useLocale, isRTL as $1697a7ad3db7b79f$re_export$isRTL} from "react-aria/I18nProvider";
 import {useLocalizedStringFormatter as $1697a7ad3db7b79f$re_export$useLocalizedStringFormatter, useLocalizedStringDictionary as $1697a7ad3db7b79f$re_export$useLocalizedStringDictionary} from "react-aria/useLocalizedStringFormatter";
 import {useListFormatter as $1697a7ad3db7b79f$re_export$useListFormatter} from "react-aria/useListFormatter";
